Python/HAT/metrics.py

```
@@ -0,0 +1,168 @@
+
+    :return: avgDist
+    :rtype: *float*
+
+    The hypergraph is clique expanded to a graph object, and the average shortest path on
+    the clique expanded graph is returned.
+    """
+    # Auth: Joshua Pickard
+    #       jpic@umich.edu
+    # Date: Nov 30, 2022
+    G = HG.cliqueGraph()
+    avgDist = nx.average_shortest_path_length(G)
```

Ruff (0.8.2)

Undefined name `nx`

(F821)